src/ImageryTab.tsx

```diff
@@ -90,14 +89,63 @@ export function ImageCard(props: {
 
 export default function ImageryTab(props: { namespace: string }) {
   const [enablePreviews, setEnablePreviews] = useState(true);
+  const [observedTopics, setObservedTopics] = useState([] as Array<string>);
+  interface ImageMap {
+    [topic: string]: ImageBitmap;
+  }
+  const [observedImages, setObservedImages] = useState({} as ImageMap);
 
-  const topics = [
-    'stereo/left/image_raw/compressed',
-    'stereo/right/image_raw/compressed',
-  ];
+  const compressedImageTopicCallback = useCallback(
+    (buf, match) => {
+      (async () => {
+        const topic = match[0] as string;
+        const ci = fb.sensor_msgs.CompressedImage.getRootAsCompressedImage(buf);
+        const blob = new Blob([ci.dataArray() ?? new Uint8Array()], {
+          type: `image/${ci.format()}`,
+        });
+
+        const bmp = await window.createImageBitmap(blob);
+
+        setObservedImages({
+          ...observedImages,
+          [topic]: bmp,
+        });
+
+        if (!observedTopics.includes(topic)) {
+          setObservedTopics([...observedTopics, topic]);
+        }
+      })();
+    },
+    [observedTopics, observedImages]
+  );
+
+  // TODO: Use a standard prefix for image topics so we only need to call this once
+  useRobofleetMsgListener(
+    matchTopic(props.namespace, 'stereo/(.*)/image_raw/compressed'),
```

Comment:
since we're arbitrarily remapping these topic names anyway, maybe we could use something simpler? Like `image/compressed/.+`?

Reply:
yeah, I was thinking about this as well. The only reason I kept the topics the same as before is that allowed me to test in real time since the jackal was up and running, publishing messages. If we change the topic names we have to go do that on our robots too, which is fine